File: 3379-score-of-a-string/score-of-a-string.js
/**
 * @param {string} s
 * @return {number}
 */
var scoreOfString = function(s) {
    let score = 0;
    for (let i = 1; i<s.length; i++){
        let diff = Math.abs(s.charCodeAt(i) - s.charCodeAt(i-1));
        score += diff;
    }
    return score
};

File: 3379-score-of-a-string/score-of-a-string.py
class Solution(object):
    def scoreOfString(self, s):
        """
        :type s: str
        :rtype: int
        """
        score = 0
        for i in range(1, len(s)):
            diff = abs(ord(s[i]) - ord(s[i - 1]))
            score += diff
        return score